roles/nuage_master/tasks/main.yaml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Install Nuage Kubemon
+  sudo: true
+  yum: name={{ nuage_kubemon_rpm }} state=present
+
+- name: set kubeconfig 
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^kubeconfig line='kubeconfig{{':'}} {{ admin_config }}'
+
+- name: set openshift admin 
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^openshiftAdmin line='openshiftAdmin{{':'}} {{ os_admin }}'
+
+- name: Set Openshift master config path
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^openshiftMasterConfig line='openshiftMasterConfig{{':'}} {{ master_config_yaml }}'
+
+- name: Set VSD API url
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^vsdApiUrl line='vsdApiUrl{{':'}} {{ vsd_api_url }}'
+
+- name: Set VSP version
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^vspVersion line='vspVersion{{':'}} {{ vsp_version }} '
+
+- name: Set enterprise 
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^enterpriseName line='enterpriseName{{':'}} {{ enterprise }}'
+
+- name: Set domain
+  sudo: true
+  lineinfile: dest={{ kubemon_yaml }} regexp=^domainName line='domainName{{':'}} {{ domain }}'
+  notify:
+    - restart master
```

Comment:
I'm assuming here that we are restarting the master because the config is being modified. I'd rather the config changes be explicitly handled through the template in openshift-master, otherwise subsequent ansible runs will result in the master config showing as changed and restarting the services again.